### ISSUE TITLE
Feature request:

### ISSUE THREAD

**Issue by jesperstarkar** [CasparCG/Client#192]
The "iNews" socket allows to push rundowns, and the dialog window for choosing and opening rundowns is nice.

With the new MySQL-support, I would love to see the ability to load rundowns from MySQL via this window. Maybe from the local sqlite as well? This way you could push static rundowns from central management without running a socket middleware with a lot of logics. If you load from database, Reload should still work, so that freshly written changes to the database loads.

**Comment by Zebiolo** [comment]
I'm not sure i follow. You mean you want to push changed rundowns to a database and reload them from there?

Custom Software -> MySQL -> Client (through reload, Ctrl+L)?

**Comment by jesperstarkar** [comment]
Exactly. I don't know why, but it seems like a nice way to manage rundowns without going through the "iNews" socket
